apps/remix-dapp/src/locales/en/udapp.json

```diff
@@ -61,6 +61,14 @@
   "udapp.reset": "Reset",
   "udapp.delete": "Delete",
   "udapp.injectedTitle": "Unfortunately it's not possible to create an account using injected provider. Please create the account directly from your provider (i.e metamask or other of the same type).",
+  "udapp.createSmartAccount": "Create safe smart account",
```

Comment:
it's either capitalized or not. you have both

Reply:
fixed